### DISCUSSION TITLE
is it possible to remap j and k keys in coc-dialog?

### DISCUSSION THREAD

**Discussion by theniceboy** [neoclide/coc.nvim#2754] [General]
![Screen Shot 2020-12-22 at 4 24 46 PM](https://user-images.githubusercontent.com/8187501/102866436-37ea6200-4472-11eb-9e82-e8ea3bac1ee4.png)
In `:CocAction` for example. Can you remap `j` and `k` to something else? This would benefit alternative keyboard layout (like Dvorak) users.

**Comment by chemzqm** [comment]
No such feature yet, it's recommended to use number key.

**Reply by theniceboy** [reply]
Thanks for your response! Number keys are good but not when the number of options exceeds 9. I think this improvement is necessary for the overall user experience of this feature.
![Screen Shot 2020-12-25 at 5 21 29 PM](https://user-images.githubusercontent.com/8187501/103129954-a1b67600-46d5-11eb-89ac-6e1c69be838d.png)

**Reply by chemzqm** [reply]
You can type numbers above 9

**Reply by theniceboy** [reply]
Actually when I type 1 the dialog box just disappears
